lib/opensusebasetest.pm

```diff
@@ -284,28 +284,23 @@ sub rewrite_static_svirt_network_configuration {
     type_line_svirt "systemctl is-active network",         expect => 'active';
 }
 
-=head2 wait_boot
+=head2 wait_for_boot_menu
 
-  wait_boot([bootloader_time => $bootloader_time] [, textmode => $textmode] [,ready_time => $ready_time] [,in_grub => $in_grub]);
+  wait_for_boot_menu([bootloader_time => $bootloader_time] [, in_grub => $in_grub]);
 
-Makes sure the bootloader appears and then boots to desktop or text mode
-correspondingly. Returns successfully when the system is ready on a login
-prompt or logged in desktop. Set C<$textmode> to 1 when the text mode login
-prompt should be expected rather than a desktop or display manager.
-C<wait_boot> also handles unlocking encrypted disks if needed as well as
-various exceptions during the boot process. Also, before the bootloader menu
-or login prompt various architecture or machine specific handlings are in
-place. The time waiting for the bootloader can be configured with
-C<$bootloader_time> in seconds as well as the time waiting for the system to
-be fully booted with C<$ready_time> in seconds. Set C<$in_grub> to 1 when the
-SUT is already expected to be within the grub menu. C<wait_boot> continues
-from there.
+Makes sure the bootloader appears and then stop in the boot menu. Only
+available on all systems where we have a detectable bootloader screen, for
```

Comment:
"Only" and "all" contradict a little bit here ;)

Reply:
so just scrap "all" I guess

Reply:
jup, would say so